lab-davide/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-davide",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FazioD/lab-06-cowsay-http-server.git"
+  },
+  "author": "faziod@uw.edu",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/FazioD/lab-06-cowsay-http-server/issues"
+  },
+  "homepage": "https://github.com/FazioD/lab-06-cowsay-http-server#readme",
+  "devDependencies": {
+    "cowsay": "^1.1.6",
```

Comment:
This should be a dependency, not a dev dependency, because it's required for your program to work. Dev dependencies are things that are needed to write and test the code, but things that are needed for the code to work ever are just regular dependencies.